mpmath/__init__.py

```diff
@@ -19,6 +19,17 @@
 fp._iv = iv
 iv._iv = iv
 
+
+def __getattr__(name):
+    if name == "dps":
+        raise AttributeError(
+            f"Module {__name__!r} does not have attribute {name!r}; "
+            "you may want 'from mpmath import mp', see Issue #657"
+        )
+    # TODO: how to pass through to access any of the stuff in here?
+    # Maintaining `__all__` would be a pain...
```

Comment:
```suggestion
    raise AttributeError(f"Module {__name__!r} has no attribute {name!r})
```

Reply:
Ah, then I have misunderstood what this does.  I take it `__getattr__` is only called when the attribute does not exist :(